### ISSUE TITLE
"Flag user" works only once.

### ISSUE THREAD

**Issue by ludwiczakpawel** [forem/forem#10605]
**Describe the bug**

When using a mod panel on `/mod` I click on a post and it reveals the preview pane where I have my mod tools available. When I close this pane and try to do the same action in another pane and I can't do it anymore.

**To Reproduce**

1. Go to `/mod`
2. Click one of the results to see the "preview pane".
3. Perform a mod action or close the preview pane.
4. Then open another preview pane and try to "Flag user" - it won't work.

TL;DR The "Flag user" modal will show up only for the first post you're trying to moderate.

**Expected behavior**

Being able to trigger "Flag user" every time.

**Desktop (please complete the following information):**

- OS, version: macOS 10.15.7
- Browser, version: Chrome 85.0.x

**Comment by Rafi993** [comment]
Hello there, if this bug is not taken I would like to help.

**Comment by ludwiczakpawel** [comment]
@Rafi993 Sweet, you've been assigned 👍

**Comment by Rafi993** [comment]
> @Rafi993 Sweet, you've been assigned 👍

Thank you @ludwiczakpawel

**Comment by Rafi993** [comment]
I was able use chrome devtools to look at event listener attached to "Flag user" button and call it manually and it worked.  Due to some reason the click event is not reaching the attached event listener after the first time.

**Comment by Rafi993** [comment]
I think this might be happening because the event listener added when the first article is opened is not being removed after it is closed.

**Comment by Rafi993** [comment]
When the event listener is called using `getEventListeners($0).click[0].listener()` it works but when the element is clicked
using ` $0.click() ` it does not work for articles after first time.

**Comment by Rafi993** [comment]
I thought this might be happening due to re-rendering of the element but that could be ruled out too.
